console/console-init/ui/src/modules/iot-device-detail/components/ConnectionGateway/ConnectionGateway.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { useParams } from "react-router";
+import {
+  Card,
+  Flex,
+  FlexItem,
+  CardBody,
+  Title,
+  Text,
+  Popover,
+  Button,
+  TextVariants,
+  CardTitle
+} from "@patternfly/react-core";
+import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons";
+import { StyleSheet, css } from "aphrodite";
+export interface IConnectionGatewayProps {
+  deviceList?: any;
+  deviceGroup?: any;
+}
+const styles = StyleSheet.create({
+  card_body: {
```

Comment:
please use rem and follow below syntax so that it can be easy to replace with different way.
Ex.
"margin-bottom": "1rem"
follow this way to add new css.